pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -610,11 +610,11 @@ type Parameter struct {
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 
 	// Default is the default value to use for an input parameter if a value was not supplied
-	Default *intstr.IntOrString `json:"default,omitempty" protobuf:"bytes,2,opt,name=default"`
+	Default *string `json:"default,omitempty" protobuf:"bytes,2,opt,name=default"`
```

Comment:
I think this value is wholly unused. Can we remove it?

Reply:
I thought that, but I actually think it's used in some weird and non-obvious dynamic way .